api/AuthenticatorAssertionResponse.json

```diff
@@ -0,0 +1,208 @@
+{
+  "api": {
+    "AuthenticatorAssertionResponse": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AuthenticatorAssertionResponse",
```

Comment:
Should this point to the main WebAuthn page?

https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API